front/src/app/shared/services/Api.service.ts

```diff
@@ -0,0 +1,24 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class ApiService {
+
+  constructor(private http: HttpClient) { }
+
+  getLists() {
+    return this.http.get<Lists>('http://localhost:3000/list/');
```

Comment:
Hardcoding the exact `host` (http://localhost:3000) is not a good idea. Read here about how to specify it in environmet.ts https://angular.io/guide/build